storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/ui/fragment/TweetsFragment.java

```diff
@@ -53,7 +53,7 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        SampleApp.get(getActivity()).getAppComponent().inject(this);
+        SampleApp.get(getActivity()).appComponent().inject(this);
```

Comment:
Because with Dagger 2 injection from Base class will inject only fields of the Base class…